app/dao/date_util.py

```diff
@@ -61,3 +61,10 @@ def get_current_financial_year_start_year():
     if now < start_date:
         financial_year_start = financial_year_start - 1
     return financial_year_start
+
+
+def which_financial_year(start_date):
```

Comment:
i don't think this file should be in the dao subfolder 🤔

Reply:
also i feel like this function should probably be called `get_financial_year_for_datetime` just to be explicit (especially about how it expects a datetime, not a date)

Reply:
update: not moving the file, dont want to make the diff of this PR too big. i did rename this function though, although it actually works fine with dates.